### ISSUE TITLE
IT Support: MOU’s Link in ROW SharePoint

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#8476]
Peter M.
ROW

Emailed - 2/11
>Paloma and the supervisors are reviewing the webpages and SharePoint.  Can you relocate the MOU’s link on the Division main page to the Dashboard page under “Resource Links”. Also, can you add an “Archives” folder within the MOU link to put old MOUs in?  Thanks!

ROW Home
![image](https://user-images.githubusercontent.com/21112975/153956223-855edd66-45b5-4865-8e15-03639b6fbde5.png)

ROW Dashboard
![image](https://user-images.githubusercontent.com/21112975/153956265-2f86e879-4089-436b-9e33-1870df14ba1a.png)

**Comment by dianamartin** [comment]
I reviewed the pages and deleted a link and created the link on the new page. I added a new folder to the document library. 

Emailed stakeholders - 2/14
>Hi there, 
Anyone who is an owner of the ROW SharePoint page can remove links or create them on whatever page you’d like. Just letting you know so you don’t have to ask and/or wait for it to be done.
>• I removed the “Special Permits and MOUs” from the main ROW SP page
>• I added a new link on the ROW Dashboard page under the “Resource Links” section and called it “Special Permits and MOUs”
![image](https://user-images.githubusercontent.com/21112975/153956456-4ec526dc-d26c-433c-b06b-c99ccb51565f.png)
>•	I added a new folder in the “Special Permits and MOUs” called “Archives”
![image](https://user-images.githubusercontent.com/21112975/153956572-93e2810b-a2af-434d-90c0-419593a5ca90.png)
